lib/chef/provider.rb

```diff
@@ -390,9 +390,6 @@ def action_#{name}
           end
         end
       end
-
-      require "chef/dsl/recipe"
-      include Chef::DSL::Recipe
```

Comment:
Like if you did `use_inline_resources` but then didn't actually use the `action` helper this would change the behavior. I'm okay with that, but it is a change I think.